client/gutenberg/extensions/publicize/connection.jsx

```diff
@@ -56,6 +69,32 @@ class PublicizeConnection extends Component {
 			</Notice>
 		);
 
+	/**
+	 * Displays a message when a connection requires reauthentication. We used this when migrating LinkedIn API usage from v1 to v2,
+	 * since the prevous OAuth1 tokens were incompatible with OAuth2.
+	 *
+	 * @returns {object|null} Notice about reauthentication
```

Comment:
Nit: We use `?` before the type to indicate nullables.

Reply:
Done.